src/router/legacyPermalinkManagement.routerPlugin.js

```diff
@@ -56,6 +56,7 @@ const handleLegacyParam = (
             if (!(projection instanceof SwissCoordinateSystem)) {
                 newValue = LV95.transformCustomZoomLevelToStandard(legacyValue)
                 if (projection instanceof CustomCoordinateSystem) {
+                    // Currently ther is no SwissCoordinateSystem which is also a CustomCoordinateSystem so this case should never be reached
```

Comment:
This would be 
// Currently, there is no CustomCoordinateSystem which is not a SwissCoordinateSystem, so this case should never be reached.